packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_phosphor_icons.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-len */
+import { Icon as PhosphorIcon } from '@phosphor-icons/react';
+import 'components/component_kit/cw_icon.scss';
+import 'components/component_kit/cw_icon_button.scss';
+import React from 'react';
+import { getClasses } from '../helpers';
+import type { IconProps, IconStyleProps } from './types';
+
+export const withPhosphorIcon = (Icon: PhosphorIcon) => (props: IconProps) => {
```

Comment:
We could probably remove it, but i believe it would make it less meaningful for what it does since it's an HOC at the end (wrapping a component with export/custom logic). 

Lmk what you think, currently its used like `withPhosphorIcon(Sparkle)`, since its just a wrapper so I think something like `PhosphorWrapper({ icon: <ICONNAME> })` might also do (or something better)!

Reply:
see comment below